index.html

```diff
@@ -0,0 +1,158 @@
+                            a significant impact in their respective fields.
+                        </p>
+                    </article>
+                </section>
+                <section>
+                    <img class="main-img" src="./assets/icons/discussion.svg" alt="">
+                    <h3 class="main-heading">Discussion</h3>
+                    <article>
+                        <p>Interactive panel discussions where experts will share their
+                            experiences, knowledge, and strategies.
+                        </p>
+                    </article>
+                </section>
+                <section>
+                    <img class="main-img" src="./assets/icons/workshop.svg" alt="">
+                    <h3 class="main-heading">Workshop</h3>
+                    <article>
+                        <p>Engaging workshops and breakout sessions that will equip
+                            you with actionable skills.
+                        </p>
+                    </article>
+                </section>
+                <section>
+                    <img class="main-img" src="./assets/icons/networking.svg" alt="">
+                    <h3 class="main-heading">Networking</h3>
+                    <article>
+                        <p>Networking opportunities to connect with a diverse community
+                            of individuals.
+                        </p>
+                    </article>
+                </section>
+                <section>
+                    <img class="main-img" src="./assets/icons/allies.svg" alt="">
+                    <h3 class="main-heading">Empower Allies</h3>
+                    <article>
+                        <p>Participants will be randomly paired up with another attendee,
+                            to engage in meaningful conversations.
+                        </p>
+                    </article>
+                </section>
+            </section>
+            <button class="main-btn">Book a Ticket</button>
+        </section>
+
+        <section class="featured-speakers">
+            <h2>Featured Speakers</h2>
+            <hr class="pink-hr">
+            <section class="featured-container">
+                <button class="more-btn">MORE <i class="fa-solid fa-chevron-down"
+                    style="color: #ec5242; margin-left: 0.5rem;"></i></button>
+            </section>
+        </section>
+        <section class="sponsors">
+            <h2>Sponsors</h2>
+            <hr class="pink-hr">
+            <section>
+                <img src="../assets/icons/mozilla.svg" alt="mozilla">
+                <img src="../assets/icons/google.svg" alt="google">
+                <img src="../assets/icons/naver.svg" alt="naver">
+                <img src="../assets/icons/daumkakao.svg" alt="daum">
+                <img src="../assets/icons/airbnb.svg" alt="airbnb">
+            </section>
```

Comment:
- [ ] For a better user experience, Kindly make sure you use the correct path for your images as you may see how they are not displaying 

![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/90b49789-d2fc-4948-b2ea-8beb3eb8270a)